N2A/src/gov/sandia/n2a/ui/settings/SettingsRepo.java

```diff
@@ -1070,17 +1067,12 @@ public void setValueAt (Object value, int row, int column)
                     if (AppData.repos.child (newName) != null) return;
                     // Now we have a legitimate name change.
 
-                    MDir models     = getModels     (oldName);
-                    MDir references = getReferences (oldName);
-                    existingModels    .remove (oldName);
-                    existingReferences.remove (oldName);
-                    existingModels    .put (newName, models);
-                    existingReferences.put (newName, references);
+                    existing.forEach ((k,v) -> v.remove (oldName));
+                    existing.forEach ((k,v) -> v.put (newName, getExisting(k, oldName)));
 
                     gitRepos.get (row).close ();  // Shut down git under oldName
                     Path repoDir = reposDir.resolve (newName);
-                    models    .set (repoDir.resolve ("models"));  // Flushes write queue, so save thread won't interfere with the move.
-                    references.set (repoDir.resolve ("references"));
+                    existing.forEach ((k,v) -> getExisting(k, oldName).set (k)); // Flushes write queue, so save thread won't interfere with the move.
```

Comment:
k here is the string name of subfolder. The original call to set took a Path object.